### PR TITLE
Minor changes of the `trigger_storage_schedule` method of the class `FlexMeasuresClient`

### PR DESCRIPTION
**Pull request by victorgarcia98**
Minor changes to the `trigger_storage_schedule` method:
* Add duration `attribute`
* Return schedule id
* Make `soc_targets` optional.

### CODE REVIEW

File: src/flexmeasures_client/client.py [hunk 1]
```diff
@@ -247,6 +249,8 @@ async def trigger_storage_schedule(
         check_for_status(status, 200)
         print("Schedule triggered successfully.")
 
+        return response.get("schedule")
```
Comment (GustaafL) [inline src/flexmeasures_client/client.py:252]: Maybe we should also set the response type to str if we are returning something now?

**Review by GustaafL — APPROVED** [review]
Looks good. Think the return of the schedule id should have been in there all along, weird that we didn't realize that before :)

File: src/flexmeasures_client/client.py [hunk 2]
```diff
@@ -208,9 +208,10 @@ async def trigger_storage_schedule(
         self,
         sensor_id: int,
         start: str | datetime,
+        duration: str | timedelta,
         soc_unit: str,
         soc_at_start: float,
-        soc_targets: list,
+        soc_targets: list = list(),
```
[the remark this answers is not in the document]
Comment (victorgarcia98) [reply src/flexmeasures_client/client.py:214]: Not really. I'll change it to `[]` to make it shorter :smile:

File: src/flexmeasures_client/client.py (hunk 1 above)
Comment (victorgarcia98) [reply src/flexmeasures_client/client.py:252]: yeah! I was going to add it but then I forgot haha

**Pull request merged**